ProjectLighthouse/Controllers/SlotsController.cs

```diff
@@ -137,18 +140,13 @@ public async Task<IActionResult> LuckyDipSlots([FromQuery] int pageStart, [FromQ
             if (token == null) return this.BadRequest();
 
             GameVersion gameVersion = token.GameVersion;
-            int slotCount = await this.database.Slots.Where(s => s.GameVersion <= gameVersion).CountAsync();
-            pageSize = Math.Min(pageSize, 30);
-
-            int skipCount = new Random().Next(seed, slotCount) + pageStart - 1;
 
-            // TODO: Incorporate seed?
             IEnumerable<Slot> slots = this.database.Slots.Where(s => s.GameVersion <= gameVersion)
                 .Include(s => s.Creator)
                 .Include(s => s.Location)
-                .Skip(skipCount)
-                .Take(pageSize)
-                .AsEnumerable();
+                .AsEnumerable() // performance?
+                .OrderBy(x => new Random().Next())
+                .Take(Math.Min(pageSize, 30));
```

Comment:
Yeah, this is definitely going to tank performance especially since we're including the `Creator` and `Location`. Looking into a server-side solution brings me to [this issue](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues/265) on the MySql EF provider which doesn't give much detail other than we need to be using `rand()` instead of `uuid()`. No details on how to use that client-side, though. Looking into this now.

Reply:
Aha! I think I got it. In EF Core 6 comes a new function named `EF.Functions.Random()` which translates into `rand()`. This should fix things.